Dockerfile

```diff
@@ -32,7 +32,7 @@ RUN curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master
 
 # Add the repos containing nginx and galaxy charts
 RUN helm repo add stable https://kubernetes-charts.storage.googleapis.com/ && \
-    helm repo add galaxy https://raw.githubusercontent.com/cloudve/helm-charts/anvil/ && \
+    helm repo add galaxy https://github.com/CloudVE/helm-charts/blob/sep29demo && \
```

Comment:
This will probably change back to the `anvil` branch -- Alex just pushed to `sep29demo` for testing + the anvil demo today @ 4.